### PR TITLE
Product page

### PR DESCRIPTION
**Pull request by tarunsankhla**
- [ ]

### CODE REVIEW

**Review by m26unkwn — APPROVED** [review]
Removed unwanted comments if it is no of use.

File: src/components/UI/Card/ProductCards/ProductCards.jsx [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+//     categoryName: "non-fiction",
+//     url:{product2},
+//     discount:"-30% off",
+// discountedPrice:3000,
+//     description:""
+//   },
+  return (
+    <>
+        <div className="card cart-card">
+                  <img className="card-img" src={productImage} alt="Food " />
```
Comment (muskanbagrecha) [inline src/components/UI/Card/ProductCards/ProductCards.jsx]: Alt can be title for better accessibility.

File: src/components/UI/Card/ProductCards/ProductCards.jsx [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+                      </div>   
+                  
+                  </div> 
+                  <div className="card-footer">
+                      <div className="card-footer-view">
+                          <button>Add to Cart</button>
+                      </div>
+                  </div>
+                  <span className="material-icons-round badge topright-badge ">
+                      favorite_border
```
Comment (muskanbagrecha) [inline src/components/UI/Card/ProductCards/ProductCards.jsx]: What is this favorite_border?

File: src/pages/WishListPage/WishListPage.jsx [hunk 3]
```diff
@@ -1,14 +1,16 @@
 import React from 'react';
 import "./WishListPage.css";
 import {
-    product1,product2,
-    product3,product4,
-    product5,product6,
-    product8,product7,
-    product9,product12,
+    product1,
+    // product2,
```
Comment (muskanbagrecha) [inline src/pages/WishListPage/WishListPage.jsx:5]: Remove unwanted comments

File: src/pages/ProductListPage/ProductListPage.jsx [hunk 4]
```diff
@@ -1,230 +1,266 @@
...
+        price: "5000",
+        categoryName: "non-fiction",
+        url:{product2},
+        discount:"-30% off",
+    discountedPrice:3000,
+        description:""
+      }]);
+    useEffect(()=>{
+        (async() => {
+        var res = await axios.get("/api/products");
```
Comment (muskanbagrecha) [inline src/pages/ProductListPage/ProductListPage.jsx:23]: Add the error handler

File: src/pages/ProductListPage/ProductListPage.jsx [hunk 5]
```diff
@@ -1,230 +1,266 @@
...
+        console.log(res);
```
Comment (muskanbagrecha) [inline src/pages/ProductListPage/ProductListPage.jsx:24]: Can remove extra console log

**Review by muskanbagrecha — APPROVED** [review]
Good work!

File: src/components/UI/Card/ProductCards/ProductCards.jsx (hunk 2 above)
Comment (tarunsankhla) [reply src/components/UI/Card/ProductCards/ProductCards.jsx]: its an icon text holder

File: src/pages/ProductListPage/ProductListPage.jsx (hunk 5 above)
Comment (tarunsankhla) [reply src/pages/ProductListPage/ProductListPage.jsx:24]: okay

File: src/components/UI/Card/ProductCards/ProductCards.jsx (hunk 1 above)
Comment (tarunsankhla) [reply src/components/UI/Card/ProductCards/ProductCards.jsx]: okay

File: src/components/Navbar/Navbar.jsx [hunk 6]
```diff
@@ -1,14 +1,15 @@
...
 import { logo as LogoImage} from "../../assets/images/Products/Products";
+import { Link, NavLink } from "react-router-dom";
 
 export function Navbar(){
     return (
         <div className="navbar">
-            <a className="project-title" href="./">
-                <img src={LogoImage}  alt="" srcset="" className="title-logo" />
-            </a>
+            <NavLink className="project-title" to="./">
```
Comment (dev-ashishsethi) [inline src/components/Navbar/Navbar.jsx:10]: Didn't know about NavLink learnt new thing from you thanks

File: src/pages/LandingPage/LandingPage.jsx [hunk 7]
```diff
@@ -1,24 +1,23 @@
 import { SlideShow } from "./SlideShow/SlideShow";
 import axios from "axios";
```
Comment (dev-ashishsethi) [inline src/pages/LandingPage/LandingPage.jsx:2]: you have imported axios you are not using it

File: src/pages/LoginPage/LoginPage.jsx [hunk 8]
```diff
@@ -0,0 +1,60 @@
...
+        {
+            var token = res.data.encodedToken;
+            localStorage.setItem("feetz",token)
+            var user = res.data.foundUser;
+            var userId =res.data.foundUser._id;
+            localStorage.setItem("feetzId",userId);
+            console.log(user,userId,token);
+            navigate("/");
+            // History.push("/products");   
+        }
```
Comment (dev-ashishsethi) [inline src/pages/LoginPage/LoginPage.jsx:24]: what will happen if the user is not found?

**Review by dev-ashishsethi — COMMENTED** [review]
please check comments

**Pull request merged**